### PR TITLE
Adding some info on the http configuration (pre-puppet)

### PR DESCRIPTION
**Pull request by djw8605**
Things will likely change when I do a puppet configuration.

### CODE REVIEW

File: docs/services/topology.md [hunk 1]
```diff
@@ -73,7 +79,15 @@ HTTPD configuration is in `/etc/httpd`; we use the modules `mod_ssl`, `mod_grids
...
-**TODO Derek can you write something here?**
+Each of the hostnames are VHosts in the apache configuration.  Some special notes:
+
+* <https://map.opensciencegrid.org> runs in the same wsgi process as the production topology, but the URL is limited to only the map code.  Further, it does not use mod_gridsite so that users are not asked to present a client certificate.
+* VHosts are configured:
+
+        :::console
+        ServerName topology.opensciencegrid.org
+        ServerAlias my.opensciencegrid.org myosg.opensciencegrid.org
+
```
Comment (brianhlin) [inline docs/services/topology.md]: I take it that the ITB hosts are configured like so?

```
ServerName topology-itb.opensciencegrid.org
ServerAlias my-itb.opensciencegrid.org
```

Comment (djw8605) [reply docs/services/topology.md]: Yes.  I just wanted to give a sample of the config.

**Comment by djw8605** [comment]
I agree map.osg.org is a bit off.  But.. it's running on the same host, in the same processes and the same namespace.  The production topology and map actually share the same data and process groups, so it doesn't have to download the data multiple times.

File: docs/services/topology.md (hunk 1 above)
Comment (brianhlin) [reply docs/services/topology.md]: I think we should include all of the necessary config here. One of the use cases I want to address with this document is "How do we spin up a new topology service if the main one goes down?"

**Comment by brianhlin** [comment]
Yea, that makes sense and this is probably a fine place for it to live now but I think that information should live in a document about Display (though leaving a note in this doc would be fine).

File: docs/services/topology.md (hunk 1 above)
Comment (djw8605) [reply docs/services/topology.md]: The total apache config for everything is on the order of ~100 lines.  Should I copy / paste everything in here?

Comment (brianhlin) [reply docs/services/topology.md]: Are you saying that the topology apache service requires ~100 lines of config on top of the default apache config? I'm more interested in the diff

Comment (djw8605) [reply docs/services/topology.md]: I was curious, so I started `wc`'ing stuff.

* To just get the production topology working: 43 non-empty, non-comment lines.
* ITB instance: 26 lines
* Map: 28 lines.

So 97 lines just in the custom configuration.  My guess was surprisingly close!

Comment (brianhlin) [reply docs/services/topology.md]: I'm ok with that. Throw the config in between details tags: https://github.com/opensciencegrid/docs/blame/master/docs/monitoring/install-rsv.md#L233-L234

Comment (brianhlin) [reply docs/services/topology.md]: @djw8605 ^^

**Comment by brianhlin** [comment]
@djw8605 I've added the PyMarkdown details extension so you can more easily add details:

https://facelessuser.github.io/pymdown-extensions/extensions/details/

**Comment by matyasselmeci** [comment]
@djw8605, I merged master and made a PR against your branch.

Isn't all the httpd config in Puppet now? I don't think we have to paste the httpd.conf in the docs anymore -- it would get out of sync.

**Comment by djw8605** [comment]
Yes, it's all in puppet.  Indeed, I may have always been in puppet.

Is there any benefit to adding it here?

**Comment by brianhlin** [comment]
Fine by me

**Pull request merged**